lib/modules/xPostLinks.js

```diff
@@ -0,0 +1,66 @@
+addModule('xPostLinks', function(module, moduleID) {
+	module.moduleName = 'X-post Links';
+	module.category = 'Posts';
+	module.description = 'Create links to x-posted subreddits in the taglines.';
+	module.include = [
+		/^https?:\/\/([a-z]+)\.reddit\.com/i
```

Comment:
This can just be `[ 'linklisting', 'comments' ]` -- since it only needs to run on pages where you see posts.